src/compas/geometry/primitives/quaternion.py

```diff
@@ -237,7 +260,7 @@ def __len__(self):
         return 4
 
     def __repr__(self):
-        return 'Quaternion({:.{prec}f}, {:.{prec}f}, {:.{prec}f}, {:.{prec}f})'.format(self.w, self.x, self.y, self.z, prec=3)
+        return "Quaternion({:.{prec}f}, {:.{prec}f}, {:.{prec}f}, {:.{prec}f})".format(self.w, self.x, self.y, self.z, prec=3)
```

Comment:
haven't we mostly settled for single quote instead of the double ones that this and other files now use?